### PR TITLE
[Application requests] Various fixes

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: tests/Controller/EnMarche/ApplicationRequestControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+                'professionalAssets' => 'my professional assets',
+                'agreeToLREMValues' => true,
+                'agreeToDataUse' => true,
+                '_token' => $form['running_mate_request[_token]']->getValue(),
+            ],
+        ], [
+            'running_mate_request' => [
+                'curriculum' => new UploadedFile(
+                    __DIR__.'/../../../app/data/files/application_requests/curriculum/cv.pdf',
+                    'cv.jpg',
```
Comment (ottaviano) [inline tests/Controller/EnMarche/ApplicationRequestControllerTest.php]: .pdf

File: tests/Controller/EnMarche/ApplicationRequestControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+                ],
+                'address' => [
+                    'address' => '8 Rue Enzo Godeas',
+                    'cityName' => 'Toulouse',
+                    'postalCode' => '31100',
+                    'city' => '31100-31555',
+                    'country' => 'FR',
+                ],
+                'favoriteCities' => ['06088', '44109'],
+                'profession' => 'developer junior',
```
Comment (ottaviano) [inline tests/Controller/EnMarche/ApplicationRequestControllerTest.php]: Tony Pong Dev junior 🤣

File: tests/Controller/EnMarche/ApplicationRequestControllerTest.php [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+use AppBundle\Repository\ApplicationRequest\ThemeRepository;
+use AppBundle\Repository\ApplicationRequest\VolunteerRequestRepository;
+use Liip\FunctionalTestBundle\Test\WebTestCase;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Tests\AppBundle\Controller\ControllerTestTrait;
+
+/**
+ * @group functional
+ */
+class ApplicationRequestControllerTest extends WebTestCase
```
Comment (ottaviano) [inline tests/Controller/EnMarche/ApplicationRequestControllerTest.php]: 💪

File: tests/Controller/EnMarche/ApplicationRequestControllerTest.php (hunk 1 above)
Comment (Remg) [reply tests/Controller/EnMarche/ApplicationRequestControllerTest.php]: you, i like you.

**Comment by thomasbisignani** [comment]
Amazing tests. Congrats. :medal_sports:

File: tests/Controller/EnMarche/ApplicationRequestControllerTest.php (hunk 1 above)
Comment (thomasbisignani) [reply tests/Controller/EnMarche/ApplicationRequestControllerTest.php]: copy-paste with the ass ?

**Pull request merged**